website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite.md

```diff
@@ -3,44 +3,134 @@ title: "Setting up SSO with Google GSuite"
+dbt Cloud Enterprise supports Single-Sign On (SSO) via Google GSuite. You will need
+permissions to create and manage a new Google OAuth2 application, as well as
+access to enable the Google Admin SDK.
 
-</Callout>
-
-dbt Cloud Enterprise supports single-sign on via Google GSuite. You will need permissions to create and manage a new Google OAuth2 application, as well as access to enable the Google Admin SDK. Currently supported features include:
+ Currently supported features include:
 
 * SP-initiated SSO
 * Just-in-time provisioning
 
-This guide outlines the setup process for authenticating to dbt Cloud with Google GSuite. After following the steps below, please contact your account manager or support (support@getdbt.com) to complete the setup process.
-
-## Configuration
-
-Log into the Google Developer Console. Create a new project or use an existing project. Navigate to **APIs & Services** &gt; **Credentials**.
-
-On the **Credentials** page you will need to create a new set of credentials to allow dbt Cloud to authenticate with your GSuite org. Click **Create Credentials**, and then select "OAuth Client ID". Select "Web Application" as the Application type.
-
-On the **Create OAuth client ID** page, enter the following:
-
-- **Name**: dbt Cloud
-- **Authorized Javascript Origins**: https://cloud.getdbt.com
-- **Authorized Redirect URIs**: https://auth.getdbt.com/login/callback
-
-Press "Create". A popup will appear with a **Client ID** and **Client Secret**. Write these down, you will need them later!
-
-<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/0ccb854-Screen_Shot_2019-12-03_at_10.11.15_AM.png" title="The 'Create Oauth client ID' page"/>
-
-Next, you will need to enable the Admin SDK for this application so that dbt Cloud can request information about your GSuite users. In the search bar, search for the "Admin SDK" API. Select it, and press "Enable" to enable it for this project.
+This guide outlines the setup process for authenticating to dbt Cloud with
+Google GSuite.
+
+## Configuration in GCP
+
+dbt Cloud uses a Client ID and Client Secret to authenticate users of a
+GSuite organization. The steps below outline how to create a Client ID and
+Client Secret for use in dbt Cloud.
+
+### Creating credentials
+
+1. Navigate to the GCP [API Manager](https://console.developers.google.com/projectselector/apis/credentials)
+1. Select an existing project, or create a new project for your API Credentials
+1. Click on **Create Credentials** and select **OAuth Client ID** in the resulting
+   popup
+1. Google requires that you configure an OAuth consent screen for OAuth
+   credentials. Click the **Configure consent screen** button to create
+   a new consent screen if prompted.
+1. On the OAuth consent screen page, configure the following settings
+    - **Application type**: internal
```

Comment:
Would it be possible to block up the settings? Depending on the screen, things format strangely.